warehouse/postgres/postgres.go

```diff
@@ -26,20 +27,32 @@ var (
 	stagingTablePrefix            string
 	pkgLogger                     logger.LoggerI
 	skipComputingUserLatestTraits bool
+	txnRollbackTimeout            time.Duration
 	connectTimeout                time.Duration
 )
 
 const (
-	host          = "host"
-	dbName        = "database"
-	user          = "user"
-	password      = "password"
-	port          = "port"
-	sslMode       = "sslMode"
-	serverCAPem   = "serverCA"
-	clientSSLCert = "clientCert"
-	clientSSLKey  = "clientKey"
-	verifyCA      = "verify-ca"
+	host                     = "host"
+	dbName                   = "database"
+	user                     = "user"
+	password                 = "password"
+	port                     = "port"
+	sslMode                  = "sslMode"
+	serverCAPem              = "serverCA"
+	clientSSLCert            = "clientCert"
+	clientSSLKey             = "clientKey"
+	verifyCA                 = "verify-ca"
+	createStagingTable       = "staging_table_creation"
```

Comment:
put these variables in a different const block just for readability ?

Reply:
done